plugins/chat/db/post_migrate/20240829140227_drop_chat_mention_notifications_old_id_column.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class DropChatMentionNotificationsOldIdColumn < ActiveRecord::Migration[7.1]
+  DROPPED_COLUMNS ||= { chat_mention_notifications: %i[old_notification_id] }
```

Comment:
```suggestion
  DROPPED_COLUMNS = { chat_mention_notifications: %i[old_notification_id] }
```

Reply:
All migrations except one use `||=` rather than `=`. In this case, I used it for consistency purposes. How strongly do you feel about this?